### PR TITLE
[IMP] website_profile: add tag when landed on profile from specific f…

### PR DESCRIPTION
**Pull request by smottet**
…orum

A tag has been added next to the title of each tab in the profile when the user come
from a specific forum. The goal is to indicate to the user that the displayed content
is specific to a given forum.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_profile/views/website_profile.xml [hunk 1]
```diff
@@ -126,7 +126,7 @@
                             <div class="row">
                                 <input name="user_id" t-att-value="user.id" type="hidden"/>
                                 <div class="form-group col-12">
-                                    <label class="text-primary mb-1 d-block" for="user_name"><span class="font-weight-bold">Real name</span></label>
+                                    <label class="text-primary mb-1 d-block" for="user_name"><span class="font-weight-bold">Name</span></label>
```
Comment (awa-odoo) [inline addons/website_profile/views/website_profile.xml:129]: Even is it's a tiny change, you should add a small note about it in your commit message.

File: addons/website_forum/views/website_forum_profile.xml [hunk 2]
```diff
@@ -100,20 +107,38 @@
...
             </t>
         </xpath>
     </template>
 
+    <template id="display_tag" name="Forum Tag">
+        <t t-if="is_unique">
+            <span class="align-items-baseline border d-inline-flex pl-2 rounded mb-1 ml-4">
+                <i class="fa fa-tag mr-2 text-muted"/>
+                <t t-esc="unique_forum_name"/>
+                <a  href="#questions" data-toggle="tab">
```
[the remark this answers is not in the document]
Comment (smottet) [reply addons/website_forum/views/website_forum_profile.xml:135]: Yeah it's a stupid thing I tried when I was desperate

[the remark this answers is not in the document]
Comment (smottet) [reply addons/website_forum/views/website_forum_profile.xml:135]: I forgot to remove it

[the remark this answers is not in the document]
Comment (awa-odoo) [reply addons/website_forum/views/website_forum_profile.xml:135]: I always re-read my own code from the PR once before greening the task.
It's very annoying, but I usually find stuff to clean / remove.

[the remark this answers is not in the document]
Comment (smottet) [reply addons/website_forum/views/website_forum_profile.xml:135]: Ok I will take this habit too.

File: addons/website_forum/views/website_forum_profile.xml [hunk 3]
```diff
@@ -100,20 +107,35 @@
...
+                            <t t-call="website_forum.forum_filter_tag"/>
+                        </h5>
                         <t t-call="website_forum.display_activities"/>
                     </div>
                 </div>
             </t>
         </xpath>
     </template>
 
+    <template id="forum_filter_tag" name="Forum Tag">
```
Comment (awa-odoo) [inline addons/website_forum/views/website_forum_profile.xml:129]: `name="Filtering Forum Tag"` ?

File: addons/website_forum/controllers/main.py [hunk 4]
```diff
@@ -553,7 +553,10 @@ def _prepare_user_profile_values(self, user, **post):
                 forums = post['forum']
             elif post.get('forum_id'):
                 forums = request.env['forum.forum'].browse(int(post['forum_id']))
-                values.update({'edit_button_url_param': 'forum_id=' + str(post['forum_id'])})
+                values.update({
+                    'edit_button_url_param': 'forum_id=' + str(post['forum_id']),
```
Comment (tde-banana-odoo) [inline addons/website_forum/controllers/main.py:557]: While we are touching this line, maybe remove string concatenation and replace it with a real formatting (forum_id=%s % blah).

File: addons/website_forum/controllers/main.py [hunk 5]
```diff
@@ -553,7 +553,10 @@ def _prepare_user_profile_values(self, user, **post):
...
+                    'filtered_on_forum': forums.name,
```
Comment (tde-banana-odoo) [inline addons/website_forum/controllers/main.py:558]: Why not namespacing it with forum before, like forum_filtered ?

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**